packages/styles/scss/components/tableofcontents/_tableofcontents.scss

```diff
@@ -103,6 +121,7 @@
         display: block;
       }
     }
+    @include themed-items;
```

Comment:
@RaphaelAmadeu you will also need to add the theme modifier class here, and not in the parent.
cc: @annawen1